tests/foreman/ui/test_audit.py

```diff
@@ -10,23 +10,22 @@
 
 """
 from fauxfactory import gen_string
-from nailgun import entities
 import pytest
 
 from robottelo.constants import ENVIRONMENT
 
 
 @pytest.fixture(scope='module')
-def module_org():
-    return entities.Organization().create()
+def module_org(module_target_sat):
+    return module_target_sat.api.Organization().create()
```

Comment:
Can we remove and use global module_org fixture?

Reply:
this PR is mostly about moving from nailgun imports to target_sat which is not the scope of this PR right now. This can be done a seperate efforts.

Reply:
Okay, but it was just removing fixture lines which are defined globally, which wouldn't impact anyway :)